front/app/component-library/components/Button/index.tsx

```diff
@@ -526,11 +525,13 @@ export interface Props extends ButtonContainerProps {
   ariaExpanded?: boolean;
   ariaPressed?: boolean;
   ariaDescribedby?: string;
+  ariaControls?: string;
   as?: React.ElementType;
   tabIndex?: number;
 }
+export type Ref = HTMLButtonElement;
 
-const Button = (props: Props) => {
+const Button = forwardRef<Ref, Props>((props, ref) => {
```

Comment:
The Button is converted to forwardRef component so that we are able to use it in a tooltip without a wrapper (otherwise there is a crash).